assets/styles.css

```diff
@@ -0,0 +1,765 @@
+  background-color: #fefaf8;
+}
+
+.more-card-details {
+  font-size: 12px;
+  line-height: 15px;
+  border-top: 1px solid #f7c9c2;
+}
+
+.more-card-details span {
+  color: #aaa;
+}
+
+.all-deals-link {
+  float: right;
+  top: -20px;
+  font-size: 0.9em;
+  color: red;
+}
+
+.slash {
+  text-decoration: line-through;
+}
+
+.deal-price {
+  font-size: 1.2em;
+  color: #333;
+}
+
+.sub-footer,
+.main-footer {
+  min-height: 30px;
+}
+
+.sub-footer {
+  background-color: #131313;
+  text-align: center;
+  padding: 30px 0;
+}
+
+.sub-footer ul {
+  padding: 10px 0;
+  margin: auto;
+}
+
+.sub-footer ul:first-child li {
+  width: 40px;
+  height: 40px;
+  border-radius: 20px;
+  margin: 0 5px;
+  background-color: #333;
+  border: solid thin #333;
+}
+
+.sub-footer ul:first-child li a {
+  top: 5px;
+}
+
+.sub-footer ul:last-child li {
+  text-transform: uppercase;
+  font-size: 11px;
+  color: #aaa;
+}
+
+.sub-footer ul:first-child li:hover {
+  background: transparent;
+}
+
+.main-footer {
+  background-color: #000;
+  font-size: 13px;
+  text-align: center;
+  padding: 30px 0;
+}
+
+.main-footer img {
+  height: 9px;
+}
+
+.copyright-text {
+  justify-content: center;
+}
+
+.copyright-text span {
+  color: #aaa;
+  padding: 5px;
+}
+
+@media screen and (min-width: 768px) {
+  .search-hamburger {
+    display: none;
+  }
+
+  .navs {
+    display: block;
+  }
+
+  .featured-articles {
+    flex-direction: row;
+    padding-top: 7rem;
+  }
+
+  .featured-article {
+    margin: 0;
+    margin-left: 10px;
+    bottom: 0;
+    min-height: 200px;
+  }
+
+  .res-aside {
+    margin-right: 10px;
+  }
+
+  .second-fa {
+    margin-bottom: 10px;
+  }
+
+  .main-art {
+    flex: 2;
+  }
+
+  .main-art .featured-article {
+    min-height: 100%;
+  }
+
+  .latest .articles {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .latest .article,
+  .article .story {
+    flex-direction: column;
+  }
+
+  .latest .article {
+    padding: 0 10px;
+  }
+
+  .story .banner {
+    min-height: 160px;
+  }
+
+  .story .text {
+    padding: 0;
+  }
+
+  .copyright-text {
+    flex-direction: row;
+  }
+
+  .latest-funding-cards {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(230px, 1fr));
+    gap: 20px;
+  }
+}
+
+/* Tablet */
+@media screen and (min-width: 768px) and (max-width: 1024px) {
+  .top-nav {
+    background-color: #131313;
+    color: #f5f5f5;
+    padding: 5px 15px;
+  }
+
+  .top-nav,
+  .main-nav {
+    justify-content: space-between;
+    font-size: 13px;
+  }
+
+  .main-nav {
+    padding: 0 15px;
+  }
+
+  .top-nav li,
+  .main-nav li,
+  .search-hamburger li {
+    display: inline-block;
+    padding: 5px;
+  }
+
+  .main-nav li {
+    padding: 15px 5px;
+    text-transform: uppercase;
+    color: lightgray;
+  }
+
+  .top-nav ul:last-child {
+    display: none;
+  }
+
+  .category {
+    flex-basis: 50%;
+  }
+
+  .latest-funding .header-title,
+  .funding-links {
+    flex-direction: column;
+  }
```

Comment:
- [ ] You did well avoiding repeated classes, the only remaining repeated rule is `flex-direction: column;` repeated in lines 127, 577 and here. Please fix that
  As a recommendation for the future, another way of doing it is using
  ```
  .flex-column {
    flex-direction: column;
  }
  ```
    And calling it in your HTML element as a class whenever you need it

Reply:
The `flex-direction: column` was repeated in those instances because they are different breakpoints from the grouped classes. Kindly study the code well and understand why we had to repeat the use on lines 577 and 644.

Thanks